cebra/solver/single_session.py

```diff
@@ -130,6 +131,16 @@ def _inference(self, batch):
 class SingleSessionHybridSolver(abc_.MultiobjectiveSolver):
     """Single session training, contrasting neural data against behavior."""
 
+    log: Dict = dataclasses.field(default_factory=lambda: ({
```

Comment:
@CeliaBenquet is this compat with your pending unified solver?

Reply:
I will revert some of these changes back, were part of the research code base and should not go to the package...

Reply:
removed, unclear why added here